### ISSUE TITLE
Rescuezilla 2.3.1 impish, USB check does not working.

### ISSUE THREAD

**Issue by Auronius** [rescuezilla/rescuezilla#306]
Good day!
When I boot Rescuezilla 2.3.1 impish from USB flash drive, and select option [Check USB for defects](https://drive.google.com/file/d/1FQMALAAB6pnpOKVj8N2CaQ8vWSi-H4nu/view?usp=sharing), it **does not checking my usb**, just loads Rescuezilla.

**Comment by Auronius** [comment]
I have downloaded 2.3.1 Focal, and yes, verification works successfully on it. So, I will use it until this bug will be fixed on Impish version.
The only question is: I have made five system backups using **2.3.1 Impish** version.  Will they restore successfully if I will use **2.3.1 Focal** version? It has older package base.

**Comment by shasheene** [comment]
Yes, backups created with Impish-based versions of Rescuezilla can be restored using Focal-based versions and vice versa
